tests/validation/cattlevalidationtest/core/test_lb.py

```diff
@@ -971,6 +971,8 @@ def test_lb_with_health_check_without_uri(client):
     delete_all(client, [lb, con1])
 
 
+@pytest.mark.skipif(
+    True, reason='round robin access check does not always apply')
```

Comment:
I don't think i should validate this use case using access check. Instead in the future , I will try validate the haproxy config in LB Agent has "leastconn" set.